deploy/helm/kubecf/values.yaml

```diff
@@ -331,12 +331,16 @@ features:
 testing:
   brain_tests:
     enabled: false
+    delete_pod: false
```

Comment:
If possible, please add documentation for this (see #736).

Why do we want this to be false by default? Not that I have a preference either way; I was just curious as to the choice.

Reply:
I made it false to avoid breaking any existing pipelines. Right now the pod is not deleted. Deleting it after completion could break a pipeline that expect the pod to be around in order to fetch the logs (e.g. the kubecf one). This also addresses @bikramnehra 's comment above.